worlds/dkc3/Client.py

```diff
@@ -116,17 +116,18 @@ async def dkc3_game_watcher(ctx: Context):
 
                 # Handle Coin Displays
                 current_level = await snes_read(ctx, WRAM_START + 0x5E3, 0x5)
-                if item.item == 0xDC3002 and (current_level[0] == 0x0A and current_level[2] == 0x00 and current_level[4] == 0x03):
+                overworld_locked = await snes_read(ctx, WRAM_START + 0x5FC, 0x1)
```

Comment:
Perhaps for readability's sake, make `overworld_locked` into `overworld_locked = (await snes_read(ctx, WRAM_START + 0x5FC, 0x1))[0] != 0x01` and use the resulting saved bool value in the elif tree.

Reply:
Sure, I'm open to that. I'll make that adjustment in a couple of hours.

Reply:
I have made the requested change.